PlaybackLab/ControlsLockScreenSampleApp/ControlsLockScreenSampleApp/Views/PlayerViewController.swift

```diff
@@ -0,0 +1,172 @@
+//
+//  PlayerViewController.swift
+//  ControlsLockScreenSampleApp
+//
+//  Copyright © 2017 Ooyala. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+import MediaPlayer
+
+class PlayerViewController: OOOoyalaPlayerViewController {
+    
+  // properties for the video
+  public var option: PlayerSelectionOption!
+  
+  // properties required for a Fairplay asset
+  private var apiKey: String?
+  private var apiSecret: String?
+  
+  // remote control center
+  private let remoteCommandCenter: MPRemoteCommandCenter = MPRemoteCommandCenter.shared()
```

Comment:
no need to add the type here on the left.

Reply:
Done